src/utils/panelFelter.js

```diff
@@ -0,0 +1,55 @@
+import * as Skjema from '../felles-komponenter/skjema';
```

Comment:
Listing av feltene i skjema gikk igjen mange steder. Jeg har løftet disse ut slik at vi bare lister feltene én gang og heller kjører reducer, map etc for å feede inn feltene i validForm der det er behov.